src/System.Private.ServiceModel/tests/Stress/SharedPoolsOfWCFObjects/Program.cs

```diff
@@ -4,14 +4,15 @@
 using System;
 using System.Threading.Tasks;
 
-namespace WCFClientStressTests
+using System.Threading;
+
+namespace SharedPoolsOfWCFObjects
 {
     public class Program
     {
         public void Main(string[] args)
         {
             ProcessRunOptions(args);
-
             DoTheRun(3000000);
```

Comment:
NIT: what does `3000000` stand for? Would much prefer a named const or something like that.